apache/superset
# feat(dashboard): Add divider component in native filters

### SUMMARY
Let users create divider component to give more context to the native filters

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
![divider-component (1)](https://user-images.githubusercontent.com/12539911/141361682-a558b7cc-285c-4603-8eef-36c1d1ab3cba.gif)


### TESTING INSTRUCTIONS
1.  Enable feature flag `DASHBOARD_NATIVE_FILTERS`
2. Go to any dashboard, and click edit (✏️) near filters.
3. Hover over Add button on the lower left of the pop-up and select divider.
4. Fill in title and description and hit save
5. See the divider in the Filter bar

### ADDITIONAL INFORMATION


- [x] Required feature flags: DASHBOARD_NATIVE_FILTERS
